tools/interop_matrix/client_matrix.py

```diff
@@ -59,4 +60,12 @@ def get_github_repo(lang):
         'v1.6.1',
         'v1.7.0',
     ],
+    'python': [
+        'v1.0.1',  #Fail to build. Investigating.
```

Comment:
Can this be changed to v1.0.x?  It looks like a build fix for Python was submitted after the v1.0.1 tag.

Reply:
@adelez Did this suggestion not work?